src/artemis/ispyb/store_in_ispyb.py

```diff
@@ -135,7 +136,24 @@ def _store_data_collection_table(self, data_collection_group_id: int) -> int:
         params["beamsize_at_samplex"] = self.ispyb_params.beam_size_x
         params["beamsize_at_sampley"] = self.ispyb_params.beam_size_y
         params["transmission"] = self.ispyb_params.transmission
-        params["comments"] = "Artemis: " + self.ispyb_params.comment
+        bottom_right = oav_utils.bottom_right_from_top_left(
+            self.upper_left,
+            self.full_params.grid_scan_params.x_steps,
+            self.y_steps,
+            self.full_params.grid_scan_params.x_step_size,
+            self.y_step_size,
+            self.ispyb_params.pixels_per_micron_x,
+            self.ispyb_params.pixels_per_micron_y,
+        )
+        params["comments"] = (
+            "Artemis: Xray centring - Diffraction grid scan of "
+            f"{self.full_params.grid_scan_params.x_steps} by "
+            f"{self.full_params.grid_scan_params.y_steps} images in "
+            f"{self.full_params.grid_scan_params.x_step_size} mm by "
+            f"{self.full_params.grid_scan_params.y_step_size} mm steps. "
+            f"Top left: [{self.upper_left.x},{self.upper_left.y}], "
+            f"bottom right: [{bottom_right.x},{bottom_right.y}]."
+        )
```

Comment:
Nit: Can you put this into a function for neatness e.g. something like `params["comments"] = self.construct_comment()`

Reply:
Yes, good idea, done